pkg/apiclient/grpcproxy.go

```diff
@@ -13,6 +13,7 @@ import (
 
```

Comment:
To avoid unnecessary conflict, please revert this file before rebasing as this was already addressed in the grpc update PR.